lib/service/AthleteApi.dart

```diff
@@ -37,4 +40,27 @@ class AthleteApi {
     print(athletesList);
     return athletesList;
   }
+
+  static Future<List<Athlete>> getAthletesFromIdsDict(
+      BuildContext context) async {
+    Map<String, List<int>> athleteIdDict = {
+      "ids": [9038, 22564, 21693, 18882],
```

Comment:
These are magic numbers if we need these as constants make them 
static final in the class and give them a meaningful name

Reply:
complete